Week1/homework/js-excercises/e5-round a number.js

```diff
@@ -0,0 +1,14 @@
+'use strict'
+
+let z =7.25;
+console.log(z);
+
+let a= Math.floor(z);
+console.log(a);
+
+let highest=[]
+if (z > a) {
+    return  console.log(highest=z);
```

Comment:
the intention of the instructions was to store the highest value first, and then log it.